### PR TITLE
Add concurrency settings for workflow

### LINKED ISSUE #267: [Feature] Add `just recreate` command
It would be nice to have the same `just recreate` command (in this case `docker-compose --file=openverse_catalog/docker-compose.yml up -d --force-recreate --build`) as we do [inside the catalog repository](https://github.com/WordPress/openverse-catalog/blob/96011a0f65a461eda78fa0cb1d864e41054b403f/justfile#L28).

### PR DESCRIPTION
**Pull request by alrz1999**
## Fixes

Related to [openverse #267](https://github.com/WordPress/openverse/issues/267#issue-1288144556) by @sarayourfriend 

## Description

This pull request Adds concurrency settings for workflow

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: .github/workflows/actionlint.yml [hunk 1]
```diff
@@ -6,6 +6,13 @@ on:
     branches:
       - main
 
+# Cancels all previous workflow runs for pull requests that have not completed.
+concurrency:
+    # The concurrency group contains the workflow name and the branch name for pull requests
+    # or the commit hash for any other events.
+    group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+    cancel-in-progress: true
+
```
Comment (sarayourfriend) [inline .github/workflows/actionlint.yml:15]: Ditto here as well: https://github.com/WordPress/openverse-frontend/pull/1881#pullrequestreview-1130808296

```suggestion
```

File: .github/workflows/ci_cd.yml [hunk 2]
```diff
@@ -65,7 +72,7 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: ${{ matrix.image }}
-          path: /tmp/${{ matrix.image }}.tar
+          path:
```
Comment (sarayourfriend) [inline .github/workflows/ci_cd.yml:75]: I think this needs to stay the same :thinking: What was the reason for this change?

```suggestion
          path: /tmp/${{ matrix.image }}.tar
```

**Review by sarayourfriend — CHANGES_REQUESTED** [review]

**Comment by krysal** [comment]
Thanks for your contribution, @alrz1999. These changes are already applied in #972.

File: .github/workflows/ci_cd.yml (hunk 2 above)
Comment (alrz1999) [reply .github/workflows/ci_cd.yml:75]: Oh, sorry. My mistake.

**Comment by krysal** [comment]
@alrz1999 Oops! You're right, my bad! I only looked at the `actionlint.yml` file, reopening it to let you rebase and finish this 😄

File: .github/workflows/actionlint.yml (hunk 1 above)
Comment (alrz1999) [reply .github/workflows/actionlint.yml]: Done

File: .github/workflows/ci_cd.yml (hunk 2 above)
Comment (alrz1999) [reply .github/workflows/ci_cd.yml]: Fixed

**Review by sarayourfriend — APPROVED** [review]
Thanks again. These look great and make sense to me. I appreciate the care you've taken to make sure each group makes sense for the context :pray:

**Review by krysal — APPROVED** [review]
Great job!

**Pull request merged**